crypto/secp256k1eth/secp256k1eth.go

```diff
@@ -8,7 +8,7 @@ import (
 	"io"
 	"math/big"
 
-	secp256k1 "github.com/btcsuite/btcd/btcec/v2"
+	secp256k1 "github.com/decred/dcrd/dcrec/secp256k1/v4"
```

Comment:
Are we sure about this change?

Reply:
did this to be in line with the changes on `crypto/secp256k1` done by PR [#4294 ](https://github.com/cometbft/cometbft/pull/4294)
any concerns with this?

Reply:
I see, then it seems like the right thing to do. Resolving